src/openforms/contrib/auth_oidc/backends.py

```diff
@@ -65,6 +65,45 @@ def get_or_create_user(
         user.is_active = True  # type: ignore
         return user
 
+    def _get_form_authentication_backend(
+        self, plugin_identifier: str
+    ) -> FormAuthenticationBackend | None:
+        return_url = self.request.session.get(_RETURN_URL_SESSION_KEY, "")
+        return_path = furl(return_url).path
+        _, _, kwargs = resolve(return_path)
+
+        try:
+            form = Form.objects.get(slug=kwargs.get("slug"))
+
+            auth_backend = FormAuthenticationBackend.objects.get(
+                form=form, backend=plugin_identifier
+            )
+        except (Form.DoesNotExist, FormAuthenticationBackend.DoesNotExist):
+            return None
+
+        return auth_backend
+
+    def _obfuscate_claims(self, claims: JSONObject):
+        # Get plugin
+        config_to_plugin = get_config_to_plugin()
+        assert self.config_class and self.config_class in config_to_plugin
+        plugin = config_to_plugin[self.config_class]
+
+        # Get authentication_backend options and make sure that they are a dict
+        authentication_backend = self._get_form_authentication_backend(
+            plugin.identifier
+        )
+        authentication_backend_options = {}
+        if authentication_backend:
+            authentication_backend_options = authentication_backend.options or {}
+
+        # Obfuscate claims using static and dynamic lists of sensitive plugin claims
+        return obfuscate_claims(
+            claims,
+            self.OIDCDB_SENSITIVE_CLAIMS
+            + plugin.sensitive_claims(authentication_backend_options, claims),
```

Comment:
please use `get_sensitive_claims` for methods that take arguments.

if it's a method that doesn't require arguments, you want to define a property instead so you can do `plugin.sensitive_claims`:

```py
@property
def sensitive_claims(self) -> Sequence[...]: ...
```